finagle-serversets/src/main/java/com/twitter/finagle/common/zookeeper/ServerSets.java

```diff
@@ -78,6 +78,6 @@ public static ServiceInstance deserializeServiceInstance(
    * @param address the target address to create the endpoint for
    */
   public static Endpoint toEndpoint(InetSocketAddress address) {
-    return new Endpoint(address.getHostName(), address.getPort());
+    return new Endpoint(address.getHostString(), address.getPort());
```

Comment:
seems good, especially about avoiding the reverse dns lookup.

i'll see about getting this merged.